.gitignore

```diff
@@ -3,6 +3,8 @@ __pycache__
 *.swp
 .tox/
 *.egg*
+*.iml
+.idea/
```

Comment:
Is it not a dupliction of the line https://github.com/zalando-incubator/kubernetes-on-aws/pull/2223/files#diff-a084b794bc0759e7a6b77810e01874f2R10 ?

Reply:
Thanks! It does look like a duplicate.